crates/abq_cli/tests/cli.rs

```diff
@@ -2908,6 +2908,7 @@ fn report_while_run_in_progress_is_error() {
 #[test]
 #[with_protocol_version]
 #[serial]
+#[ignore]
 fn test_explicit_run_id_against_ephemeral_queue() {
```

Comment:
This definitely passes for me locally 🤔

Reply:
Does it still fail on CI with the changes I pushed Friday?

Reply:
Checking and will update accordingly :)